dhcpv4/nclient4/lease.go

```diff
@@ -18,17 +18,17 @@ type Lease struct {
 	CreationTime time.Time
 }
 
-// Release send DHCPv4 release messsage to server, based on specified lease.
-// release is sent as unicast per RFC2131, section 4.4.4.
-// Note: some DHCP server requries of using assigned IP address as source IP,
-// use nclient4.WithUnicast to create client for such case.
+// Release sends a DHCPv4 release message to server, based on the specified lease.
```

Comment:
Same: All these comment updates are unrelated to the RENEW support. Maybe they should be a different PR.